JOSESwift/Sources/Algorithms.swift

```diff
@@ -30,7 +30,11 @@ import Foundation
 /// - ES512: [ECDSA P-521 using SHA-512](https://tools.ietf.org/html/rfc7518#section-3.4)
 public enum SignatureAlgorithm: String {
     case RS256
+    case RS384
     case RS512
+    case PS256
```

Comment:
``` suggestion
    @available(iOS 11, *) case PS256
```

It's great that you check the API availability of the RSASSA-PSS algorithms in RSA.swift!

Please also tag the respective cases here as `@available(iOS 11.0, *)`. This will make it much easier for users to figure out whether the APIs are available to them or not.

Reply:
Ah... That is a lot better :) These are the first lines of Swift I ever wrote so I was basically Googling for Swift and #ifdef ;)

Reply:
No worries, and welcome to the Swift community! 🎉